darwin.yml

```
@@ -0,0 +1,10 @@
+version: "3"
+
+services:
+  db:
+    ports:
+      - ${DB_PORT:-5555}:5432
```

```suggestion
      - ${DB_PORT:-5432}:5432
```
 If the `.env.dev` is fill it will be used 💪